### PR TITLE
prepare release 0.1.0

### PR DESCRIPTION
**Pull request by fneum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,103 @@
...
-The [PyPSA-Eur](https://github.com/PyPSA/pypsa-eur) cutout as has to be placed in ``input_files/cutouts``. After running the snakemake workflow, the output ``.csv`` file will be located in ``output/energy_demand/``.
-
-## Installation
+# Interannual Electricity Demand Calculator
+
+Large parts of this code were originally developed by [Lieke van der
+Most](https://github.com/L-vdM) (University of Groningen) in the *EU renewable
+energy modelling framework*. The original version of the code can be found
+[here](https://github.com/L-vdM/EU-renewable-energy-modelling-framework) and is
+referenced below as [1]. This model has been validated against historical
```
Comment (martacki) [inline README.md:7]:
```suggestion
referenced below as [1]. [1] has been validated against historical
```

File: README.md [hunk 2]
```diff
@@ -1,67 +1,103 @@
...
+```
 
-    mamba env create -f envs/environment.yaml
+After successfully running the workflow, the output files will be located in
+`output/energy_demand` named `demand_hourly_{yr}.csv`.
 
+The years to compute can be modified directly in the `Snakefile`.
 
-# Example Output
+## Example Outputs
```
Comment (martacki) [inline README.md:93]:
```suggestion
## Example Plots of the Workflow Output
```

File: README.md [hunk 3]
```diff
@@ -1,67 +1,103 @@
...
 
-An exemplary result for Germany (DE) plot of an exemplary week of January, 2013 comparing the results of this workflow with the Open Power System Data.
+An exemplary result for Germany (DE) plot of an exemplary week of January, 2013
+comparing the results of this workflow with the Open Power System Data.
```
Comment (martacki) [inline README.md:96]:
```suggestion
An exemplary plot of the result for Germany (DE) of an exemplary week of January, 2013
comparing the results of this workflow with the [Open Power System Data](https://data.open-power-system-data.org/time_series/) in hourly resolution.
```

File: README.md [hunk 4]
```diff
@@ -1,67 +1,103 @@
...
 
 ![ts-DE](https://user-images.githubusercontent.com/53824825/188666599-bff05561-e601-40d0-9e90-51a6eb68455c.png)
 
-An exemplary result for Spain (ES) plot of an exemplary week of January, 2013 comparing the results of this workflow with the Open Power System Data.
+An exemplary result for Spain (ES) plot of an exemplary week of January, 2013
+comparing the results of this workflow with the Open Power System Data.
```
Comment (martacki) [inline README.md:101]:
```suggestion
An exemplary plot of the result for Spain (ES) of an exemplary week of January, 2013
comparing the results of this workflow with the [Open Power System Data](https://data.open-power-system-data.org/time_series/) in hourly resolution.
```

**Comment by martacki** [comment]
Thanks @fneum for careful preparation!

File: model/constants/mappings.py [hunk 5]
```diff
@@ -1,52 +1,295 @@
...
-     ]
-names = ['Albania', 'Austria', 'Ukraine', 'Bosnia and Herzegovina', 'Belgium', 'Bulgaria', 
-         'Switzerland', 'Cyprus', 'Czechia', 'Germany', 'Denmark', 'Estonia', 'Greece', 'Spain',
-         'Finland', 'France', 'Croatia', 'Hungary', 'Ireland', 'Iceland', 'Italy', 
-         'Liechtenstein', 'Lithuania', 'Luxembourg', 'Latvia', 'Moldova, Republic of', 
-         'Montenegro', 'Macedonia, the former Yugoslav Republic of', 'Malta', 'Netherlands', 
-         'Norway', 'Poland', 'Portugal', 'Romania', 'Serbia', 'Sweden', 'Slovenia', 'Slovakia', 
-         'Turkey', 'United Kingdom', 'United Kingdom', 'Kosovo'
-        ]
+l0 = [
```
[the remark this answers is not in the document]
Comment (fneum) [reply model/constants/mappings.py:7]: well if I were to redo this, yes

**Comment by fneum** [comment]
Great, all suggested changes applied. Ready to go from my side.

**Review by martacki — APPROVED** [review]

**Pull request merged**